src/Features/TaskTemplates/Sidenav/AddTaskTemplate/AddTaskTemplate.tsx

```diff
@@ -43,12 +43,20 @@ function AddTaskTemplate({ addTemplateInState, taskTemplates, history, location
       closeModal();
     } catch (err) {
       if (!isCancel(err)) {
+        let errorTitle = err.toString();
```

Comment:
We have a `formatErrorMessage` util, did yall try using that for formatting the error? 

`import { formatErrorMessage } from "@boomerang-io/utils";`